pyutilib/th/pyunit.py

```diff
@@ -294,6 +294,10 @@ def recordTestData(self, name, value):
         if not tmp is None:
             tmp[name] = value
 
+    def shortDescription(self):
+        # Disable nose's use of test docstrings for the test description.
```

Comment:
Is this shortDescription used for anything but nose? Just checking to make sure were aren't breaking some other potential use of this function.

Reply:
Not to my knowledge.  This is the base class for test suites, so it is not used outside of the testing framework.

Reply:
OK. Sounds good. Good change.